__snapshots__/flex-status.test.js

```diff
@@ -163,4 +71,96 @@ exports['flex status by specifying credentials as options when invalid and exist
 [debug] Response: POST http://localhost:3234/session 401
 [error] InvalidCredentials: Credentials are invalid. Please authenticate.
 
+`
+
+exports['flex status by specifying a profile and existent serviceId should succeed and output default format 1'] = `
+[debug] Checking for package updates
+[debug] Using profile 'profileToGetServiceStatus'
+[debug] Project configuration file not found: 'projectSetupPath'.
+[debug] Request:  GET http://localhost:3234/v2/data-links/12378kdl2/status
+[debug] Response: GET http://localhost:3234/v2/data-links/12378kdl2/status 200
+key            value                                
+-------------  -------------------------------------
+status         ONLINE                               
+version        1.4.2                                
+id             12378kdl2                            
+requestedAt    
```

Comment:
I think it would be better if the date is replaced by something different than an empty string, so that if you take a look at the snapshot you won't think something has gone wrong. It could be 'someDate' or whatever you think suitable.